client/keys/rename_test.go

```diff
@@ -39,15 +41,15 @@ func Test_runRenameCmd(t *testing.T) {
 
 	clientCtx := client.Context{}.
 		WithKeyringDir(kbHome).
-		WithKeyring(kb)
+		WithCodec(cdc)
```

Comment:
Since old test logic yield an [error](https://play.golang.org/p/DijWsX0iCEN), I would prefer to keep `withCodec(cdc)` . In many cases we apply ` flags.FlagKeyringBackend` on `cmd`. Hence, `NewKeyringFromBackend(clientCtx, keyringBackend)` is used to create a `keyring` which uses `ctx.Codec` under the hood.